signac/core/utility.py

```diff
@@ -3,8 +3,12 @@
 # This software is licensed under the BSD 3-Clause License.
 import re
 import subprocess
+from deprecation import deprecated
+from ..version import __version__
 
 
+@deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+            details="This function is obsolete, use equivalent function in connection module.")
```

Comment:
Since the `connection` module is also deprecated, I advise that we just say it's obsolete.